### DISCUSSION TITLE
v0.13.0-beta.2

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#829] [Releases]
## What's Changed
* [Bugfix] Missing Filament assets by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/827
* Release v0.13.0-beta.2 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/828

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.13.0-beta.1...v0.13.0-beta.2

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.13.0-beta.2'>v0.13.0-beta.2</a>.</em>

**Comment by jiriteach** [comment]
Any way of changing "Dashboard" to pick up the Title? or Name?

<img width="1532" alt="Screenshot 2023-10-03 at 09 13 32" src="https://github.com/alexjustesen/speedtest-tracker/assets/2116446/9178b1b1-cde0-4035-bf84-62b51fc2133d">

**Reply by alexjustesen** [reply]
Could do the same as the title really easy so it would match what's in your tab title.

**Reply by jiriteach** [reply]
Awesome - that would make sense. Thanks

**Reply by alexjustesen** [reply]
Done, working on releasing `-beta.3` now.

https://github.com/alexjustesen/speedtest-tracker/pull/832

**Reply by jiriteach** [reply]
That great! - Thanks for the quick turnaround :)
